app/pathogen/arbovirus/dashboard/(visualizations)/visualizations-section.tsx

```diff
@@ -0,0 +1,70 @@
+import { useCallback, useContext } from "react";
+import { useRouter } from "next/navigation";
+import { VisualizationId, VisualizationInformation, addToVisualizationInformation } from "../../visualizations/visualizations";
+import { ZoomIn } from "lucide-react";
+import { cn } from '@/lib/utils';
+import { ArbovirusPageSectionId } from "../../../../constants";
+import { ArboContext } from "@/contexts/arbo-context";
+import { useArboDataInsights } from "@/hooks/useArboDataInsights";
+
+export const VisualizationsSection = () => {
+  const { filteredData } = useContext(ArboContext);
+  const { getNumberOfUniqueValuesForField } = useArboDataInsights();
+
+  const areLessThanTwoWHORegionsPresentInData = getNumberOfUniqueValuesForField({
+    filteredData: filteredData.filter((dataPoint) => !!dataPoint.whoRegion),
+    fieldName: 'whoRegion'
+  }) < 2;
+
+  const router = useRouter();
+  const allVisualizationInformationWithClassnames = addToVisualizationInformation({
+    [VisualizationId.MEDIAN_SEROPREVALENCE_BY_WHO_REGION_AND_AGE_GROUP]: { className: "h-full" },
+    [VisualizationId.ESTIMATE_COUNT_BY_WHO_REGION_AND_ARBOVIRUS]: { className: "h-1/2" },
+    [VisualizationId.ESTIMATE_COUNT_BY_UN_REGION_AND_ARBOVIRUS]: { className: "h-1/2" },
+    [VisualizationId.ESTIMATE_COUNT_BY_ARBOVIRUS_AND_ANTIBODY_TYPE ]: { className: "h-1/2" },
+    [VisualizationId.MEDIAN_SEROPREVALENCE_BY_WHO_REGION]: { className: "h-full" },
+    [VisualizationId.CUMULATIVE_ESTIMATE_COUNT_OVER_TIME_BY_ARBOVIRUS]: { className: "h-1/2" },
+    [VisualizationId.CUMULATIVE_ESTIMATE_COUNT_OVER_TIME_BY_SAMPLE_FRAME]: { className: "h-full 2xl:h-3/4" },
+    [VisualizationId.TOP_TEN_COUNTRIES_REPORTING_ESTIMATES_BY_ARBOVIRUS]: { className: "h-full" },
+    [VisualizationId.CHANGE_IN_MEDIAN_SEROPREVALENCE_OVER_TIME]: { className: "h-full" },
+  })
+
+  const visualizationsOnLeftSide = allVisualizationInformationWithClassnames.filter((visualizationInfo) => [
+    areLessThanTwoWHORegionsPresentInData ? VisualizationId.ESTIMATE_COUNT_BY_UN_REGION_AND_ARBOVIRUS : VisualizationId.ESTIMATE_COUNT_BY_WHO_REGION_AND_ARBOVIRUS,
```

Comment:
Line 33 new

Basically when I did this: 
I forgot the WIP page and only did the dashboard page. Whoopsies. I also re-arranged the order of the graphs a bit here to match what they should have been.